RevenueCatUI/Data/PaywallTemplate.swift

```diff
@@ -23,6 +23,9 @@ enum PaywallTemplate: String {
     case template4 = "4"
     case template5 = "5"
     case template7 = "7"
+    #if PAYWALL_COMPONENTS
+    case templateComponents = "components"
+    #endif
```

Comment:
@jamesrb1 Does this case even need to be here anymore? 🤔 We are doing our paywall components logic in `PaywallView` before any of the `PaywallData` is being processed so `PaywallData` should never know about this kind of template 🤷‍♂️  I think we can get rid of this case and all to code associated with it in `PaywallData`

Reply:
Gone!